docs/tech/python-packages.md

```diff
@@ -0,0 +1,40 @@
+# Python packages
+
+A lot of our code is published as Python packages on [PyPI][].
+
+[PyPI]: https://pypi.org/
+
+## Development
+
+Make changes to the repository that contains the Python package using
+the normal GitHub flow.
+
+Bump the version in the repository. You might have to look in:
+
+- `setup.py`
+- `package/__init__.py`
+- `docs/conf.py`
+
+## Release
+
+Use [`git flow release`](../tools.html#git-flow) to publish a new release
+to GitHub. This will create a new tag which will cause Travis to build.
```

Comment:
There is probably more we could write here... `develop` branch is usually the default branch, PRs target that branch, `master` tracks the latest releases, etc. But maybe that should be in a different PR? Or maybe people should just read on the git flow page? Not sure how obvious this is.

Also, `git flow release` doesn't push anything, you still have to push the branches and tag yourself.